src/platform/common/utils.node.ts

```diff
@@ -29,45 +23,3 @@ export async function tryGetRealPath(expectedPath: Uri): Promise<Uri> {
         return expectedPath;
     }
 }
-
-export async function calculateWorkingDirectory(
-    configService: IConfigurationService,
```

Comment:
This code doesn't belong here, this is very specific to kernels hence should not be under `platnform/common `